### PR TITLE
Enable login_SUITE for dynamic domains

### PR DESCRIPTION
**Pull request by Premwoik**
Enable login_SUITE for dynamic domains.

### CODE REVIEW

**Comment by mongoose-im** [comment]
[small_tests_24](https://circleci.com/gh/esl/MongooseIM/73443) / small_tests / 027752779d2539ac7d901817041b01afadfc89f0
Reports [root](http://esl.github.io/circleci-mim-results/s3_reports.html?prefix=PR/3278/73443/small_tests.24.0.5) / [small](https://circleci-mim-results.s3.eu-central-1.amazonaws.com/PR/3278/73443/small_tests.24.0.5/small/ct_run.mongooseim@localhost.2021-09-16_15.25.35/lib.mongooseim.logs/run.2021-09-16_15.25.35/suite.log.html)

**Comment by mongoose-im** [comment]
[small_tests_24](https://circleci.com/gh/esl/MongooseIM/73698) / small_tests / a0a57e8166873ee8ea0c9a05c954aaa4525f6bab
Reports [root](http://esl.github.io/circleci-mim-results/s3_reports.html?prefix=PR/3278/73698/small_tests.24.0.5) / [small](https://circleci-mim-results.s3.eu-central-1.amazonaws.com/PR/3278/73698/small_tests.24.0.5/small/ct_run.mongooseim@localhost.2021-09-17_07.31.54/lib.mongooseim.logs/run.2021-09-17_07.31.54/suite.log.html)

**Review by chrzaszcz — COMMENTED** [review]
Looks good, one minor comment from me.

File: big_tests/dynamic_domains.config [hunk 1]
```diff
@@ -146,7 +146,21 @@
...
+    {neustradamus, [
+        {username, <<"neustradamus">>},
+        {server, <<"domain.example.com">>},
+        {host, <<"localhost">>},
+        {password, <<"scramshaplus">>},
+        {starttls, required},
+        {tls_module, fast_tls}]},
+    {hacker, [
+        {username, <<"hacker">>},
+	{server, <<"domain.example.com">>},
```
[the remark this answers is not in the document]
Comment (Premwoik) [reply big_tests/dynamic_domains.config:159]: When I check the file, the indentation looks good

[the remark this answers is not in the document]
Comment (vkatsuba) [reply big_tests/dynamic_domains.config:159]: Issue in using tabs and spaces, eg in line 159 you use tab, in 157 you use 4 spaces. Please remove tab and add spaces - this should help.

**Comment by mongoose-im** [comment]
[small_tests_24](https://circleci.com/gh/esl/MongooseIM/74071) / small_tests / 641fab5ca6f52e318a88fde30bf33b823beb3d0c
Reports [root](http://esl.github.io/circleci-mim-results/s3_reports.html?prefix=PR/3278/74071/small_tests.24.0.5) / [small](https://circleci-mim-results.s3.eu-central-1.amazonaws.com/PR/3278/74071/small_tests.24.0.5/small/ct_run.mongooseim@localhost.2021-09-17_10.24.02/lib.mongooseim.logs/run.2021-09-17_10.24.03/suite.log.html)

**Review by vkatsuba — APPROVED** [review]

**Review by chrzaszcz — APPROVED** [review]
Looks good!

**Pull request merged**